File: Java/Basic/Acos.java
import java.util.*;
import java.lang.*;
import java.io.*;
 
class Main
{
    public static void main( String[] args ) {
    	double deg = 30;
    	double rad = Math.toRadians(deg);
		double ans = Math.acos(rad);
        System.out.println(ans);
    }
}

File: CSharp/Basic/Acos.cs
using System;

namespace Test
{
	class Program
	{
		static void Main(string[] args)
		{
			double deg = 30;
			double rad = System.Math.PI * deg / 180.0;
			double y = System.Math.Acos(rad);
			Console.WriteLine("計算結果：" + y);
		}
	}
}
